src/Microsoft.FeatureManagement.AspNetCore/DefaultHttpTargetingContextAccessor.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.AspNetCore.Http;
+using Microsoft.FeatureManagement.FeatureFilters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Provides a default implementation of <see cref="ITargetingContextAccessor"/> that creates <see cref="TargetingContext"/> using info from the current HTTP request.
+    /// </summary>
+    public class DefaultHttpTargetingContextAccessor : ITargetingContextAccessor
```

Comment:
Maybe could be `sealed` since there's no member to override, it would be fine from `sealed` to non-sealed but breaking change from non-sealed to `sealed`

```suggestion
    public sealed class DefaultHttpTargetingContextAccessor : ITargetingContextAccessor
```